pcdsdevices/slits.py

```diff
@@ -602,6 +602,18 @@ def calc_lightpath_state(
                                             ywidth=ywidth_readback)
 
 
+class SL2K4Slits(BeckhoffSlits):
+    def __init__(self, prefix, cam, data_source, *, name, **kwargs):
+        self._started_move = False
+        self._top_done = False
+        self._bottom_done = False
+        self._north_done = False
+        self._south_done = False
```

Comment:
Question/confusion: Aren't these all already ran in the `super().__init__` call? Are they needed again here?